### PR TITLE
Minor cleanups to emscripten.py

### PR DESCRIPTION
**Pull request by sbc100**
- use """ for docstrings.
- don't overuse default args.  In this case they are always specified.
- libaries can never be None so don't check this.

### CODE REVIEW

File: emscripten.py [hunk 1]
```diff
@@ -2276,17 +2265,19 @@ def main(args, compiler_engine, cache, temp_files, DEBUG):
              temp_files=temp_files, DEBUG=DEBUG)
 
 
-def _main(args=None):
-  if args is None:
-    args = sys.argv[1:]
+def _main():
```
Comment (jgravelle-google) [inline emscripten.py]: This probably breaks when invoking `emcc`.

`emcc.py` calls `shared.Building.emscripten(...)` which does:
```python
    from emscripten import _main as call_emscripten
    settings = Settings.serialize()
    args = settings + extra_args
    cmdline = [filename + ('.o.ll' if append_ext else ''), '-o', filename + '.o.js'] + args
    with ToolchainProfiler.profile_block('emscripten.py'):
      call_emscripten(cmdline)
```

I'd suggest leaving this with the `args` argument and invoking `_main` down below as `sys.exit(_main(sys.argv[1:]))`, which secretly I like better anyway

File: emscripten.py [hunk 2]
```diff
@@ -2276,17 +2265,19 @@ def main(args, compiler_engine, cache, temp_files, DEBUG):
...
+  if os.environ.get('EMCC_FAST_COMPILER') == '0':
```
Comment (kripken) [inline emscripten.py]: actually we can just remove these 3 lines about `EMCC_FAST_COMPILER` - that's warning about something we removed 3 years ago or more, at this point.

Comment (sbc100) [reply emscripten.py]: Done

Comment (kripken) [reply emscripten.py]: thanks, lgtm too

**Pull request merged**